lib/aws-xray-sdk/lambda/lambda_recorder.rb

```diff
@@ -0,0 +1,19 @@
+require 'aws-xray-sdk/recorder'
+
+module XRay
+  class LambdaRecorder < Recorder
+    include Logging
+
+    def begin_segment(name, trace_id: nil, parent_id: nil, sampled: nil)
```

Comment:
Not sure if this works, since I'm able to create a segment using `XRay.recorder.begin_segment 'my_segment`. Can you confirm this as well?

Reply:
Added test to demo project, and seems to be working as expected ( logs warning, returns current Facade Segment)

Reply:
I see. Some problem on my end.